src/main/python/vantiqsdk.py

```diff
@@ -1336,6 +1337,15 @@ async def ack(self, request_id: str, subscription_id: str, msg: dict) -> None:
         await self._subscriber.ack(request_id, subscription_id, sequence_id, partition_id)
         return
 
+    def register_subscriber_on_close(self, callback: Callable[[], Awaitable[None]]):
```

Comment:
The LCEL host uses this to cleanup on disconnect (transient or not, it doesn't matter since the server will nuke the state for the previous WS connection anyway).